package.json

```diff
@@ -80,5 +80,8 @@
       "<rootDir>/example/node_modules",
       "<rootDir>/lib/"
     ]
+  },
+  "dependencies": {
+    "react-native-icomoon": "^0.1.1"
```

Comment:
I know that for some library like react-native-paper (often library called 'react-native-something') that has some native code that need to be link, we needed to install it as a peerdependencies. Dont know if it the case here with react-native-icomoon